pkg/process/checks/container.go

```diff
@@ -119,6 +119,12 @@ func (c *ContainerCheck) Run(nextGroupID func() int32, options *RunOptions) (Run
 	if len(containers)%c.maxBatchSize != 0 {
 		groupSize++
 	}
+
+	// For no chunking manual checks, set groupsize as 1 to ensure one chunk
```

Comment:
Let's remove the wording about manual checks in the checks themselves. The reason we disable chunking is because of manual check subcommand, but that info should remain there. From the perspective of the check we just know the caller requested it to disable chunking and this is how it happens.